test/runtests.jl

```diff
@@ -1,15 +1,14 @@
 using MakieGallery, AbstractPlotting, GLMakie, Test
 using MakieGallery: @block, @cell
-empty!(MakieGallery.plotting_backends)
```

Comment:
Any particular reason to remove these?

Reply:
I kinda deprecated this ;)

Reply:
I needed it for external package support - MakieGallery (surprisingly) works with Plots.jl with minimal tweaking, so it would be nice if we could keep it that way...

Reply:
Oh...how are you actually using it?

Reply:
```julia
empty!(MakieGallery.plotting_backends)
push!(MakieGallery.plotting_backends, "Plots")

function MakieGallery.save_media(example, plot::Plots.Plot, path::String)
    plotpath = joinpath(path, "image.svg")
    Plots.savefig(plot, plotpath)
    return [plotpath]
end
```
and it just works.

Reply:
So...are you running the normal makie examples like that?Or your own?

Reply:
I've defined my own set of examples using Plots (and some other packages), which I then run using this.  Regression testing isn't really a concern for me, so SVG output is fine.

Reply:
So can you just say `using Plots` ?

Reply:
In the toplevel code, yes, but then it gets mixed in with `using Makie` as well.